pkg/store/artifactory_store_test.go

```
@@ -42,21 +41,21 @@
 	targetDir := params[0].Target
 	filename := filepath.Base(params[0].Pattern)
 
-	if err := os.MkdirAll(targetDir, 0755); err != nil {
+	if err := os.MkdirAll(targetDir, 0o755); err != nil {
 		return 0, 0, err
 	}
 
 	data := []byte(`{"test":"value"}`)
 	fullPath := filepath.Join(targetDir, filename)
-	if err := os.WriteFile(fullPath, data, 0644); err != nil {
+	if err := os.WriteFile(fullPath, data, 0o644); err != nil {
```

golangci-lint (1.62.2)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)